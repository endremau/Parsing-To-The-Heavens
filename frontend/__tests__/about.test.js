/*
This file is part of SKRP.

SKRP is free software: you can redistribute it and/or modify
it under the terms of the GNU Lesser General Public License as published
by the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

SKRP is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU Lesser General Public License for more details.

You should have received a copy of the GNU Lesser General Public License
along with SKRP. If not, see <https://www.gnu.org/licenses/>.
*/
import React from 'react';
import renderer from 'react-test-renderer'
import About from '../src/components/about'

it('renders correctly', ()=>{
    const tree=renderer
        .create(<About/>)
        .toJSON();
    expect(tree).toMatchSnapshot()
})