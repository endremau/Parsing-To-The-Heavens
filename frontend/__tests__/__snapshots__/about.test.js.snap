// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="sc-bdVaJa kjxJpE"
    style={
      Object {
        "border": "none",
      }
    }
  >
    <h1
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      SKRP
    </h1>
    <br />
    <span>
      <p>
        localmain
      </p>
    </span>
  </div>
  <div
    className="sc-bdVaJa kjxJpE"
  >
    <h1
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Backend
    </h1>
    <br />
    <span>
      <p>
        localbackend
      </p>
    </span>
  </div>
  <div
    className="sc-bdVaJa kjxJpE"
  >
    <h1
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Mocker
    </h1>
    <br />
    <span>
      <p>
        localmocker
      </p>
    </span>
  </div>
  <div
    className="sc-bdVaJa kjxJpE"
  >
    <h1
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      Frontend
    </h1>
    <br />
    <span>
      <p>
        localfrontend
      </p>
    </span>
  </div>
  <div
    className="sc-bdVaJa kjxJpE"
    style={
      Object {
        "border": "none",
        "textAlign": "center",
      }
    }
  >
    <i>
       Used the 
       version of the docs
    </i>
  </div>
</div>
`;
